### PR TITLE
add feature for globe

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

**Comment by 3nids** [comment]
@m-kuhn  is this enough?

**Comment by m-kuhn** [comment]
Can we also have a CMake update to request this feature depending on config option

```
if(WITH_GLOBE)
  list(APPEND VCPKG_MANIFEST_FEATURES "globe")
endif()
```

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,6 +19,10 @@ SET(KADAS_VERSION_PATCH "2")
 SET(KADAS_VERSION_PRE "")
 EXECUTE_PROCESS(COMMAND git rev-parse HEAD OUTPUT_VARIABLE KADAS_VERSION_DEV OUTPUT_STRIP_TRAILING_WHITESPACE)
 
+if(WITH_GLOBE)
```
Comment (m-kuhn) [inline CMakeLists.txt:22]: this needs to be before the `include(VcpkgToolchain)` or it won't be respected

File: CMakeLists.txt [hunk 2]
```diff
@@ -6,6 +6,11 @@ SET(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 # TODO: Is that actually used?
 INCLUDE(GenerateExportHeader)
 
+SET(WITH_GLOBE ON CACHE BOOL "Enable globe")
```
Comment (m-kuhn) [inline CMakeLists.txt:9]:
```suggestion
set(WITH_GLOBE ON CACHE BOOL "Enable globe")
```

I guess we'll do a batch update of style once the vcpkg stuff is settled

**Comment by 3nids** [comment]
okay?

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**